ocamltest/tsl_parser.mly

```diff
@@ -24,29 +24,55 @@ let mkstring s = make_string ~loc:(symbol_rloc()) s
 
 let mkidentifier id = make_identifier ~loc:(symbol_rloc()) id
 
-let mkenvstmt envstmt =
+ let mkenvstmt envstmt =
   let located_env_statement =
     make_environment_statement ~loc:(symbol_rloc()) envstmt in
   Environment_statement located_env_statement
 
 %}
 
-%token TSL_BEGIN_C_STYLE TSL_END_C_STYLE
-%token TSL_BEGIN_OCAML_STYLE TSL_END_OCAML_STYLE
-%token COMMA
+%token <bool> TSL_BEGIN_C_STYLE
+%token TSL_END_C_STYLE
+%token <bool> TSL_BEGIN_OCAML_STYLE
+%token TSL_END_OCAML_STYLE
+%token COMMA OPEN CLOSE SEMI
 %token <int> TEST_DEPTH
 %token EQUAL PLUSEQUAL
 /* %token COLON */
 %token INCLUDE SET UNSET WITH
 %token <string> IDENTIFIER
 %token <string> STRING
 
-%start tsl_block
-%type <Tsl_ast.tsl_block> tsl_block
+%start tsl_block_old tsl_block
+%type <Tsl_ast.tsl_block> tsl_block_old
+%type <Tsl_ast.t> tsl_block
```

Comment:
I find the name `tsl_block` confusing, the symbols could be called `tsl_script` and `tsl_script_old` instead.

Reply:
I've reverted the old one to its original `tsl_block` (which will go away soon) and used `tsl_script` for the new syntax.